src/test/java/com/cloudbees/jenkins/plugins/advisor/AdvisorGlobalConfigurationTest.java

```diff
@@ -220,8 +218,8 @@ public void testSaveExcludedComponents() throws Exception {
     j.executeOnServer(doConfigure);
     assertThat(advisor.getExcludedComponents().size(), is(25));
     HtmlPage managePage = wc.goTo("cloudbees-jenkins-advisor");
-    assertTrue(managePage.asText().contains("uncheckedJenkinsLogs"));
-    assertTrue(managePage.asText().contains("uncheckedSlaveLogs"));
+    assertTrue(managePage.asNormalizedText().contains("unchecked JenkinsLogs"));
```

Comment:
I came up with much the same patch, except replacing e.g.
```suggestion
    assertThat(managePage.asNormalizedText(), containsString("unchecked JenkinsLogs"));
```

Reply:
Not going to bother with an unrelated Hamcrest cleanup, trying to keep it simple.

Reply:
Yeah I did that mainly because the test failure I got initially did not say what the page _did_ contain!